kong/plugins/moesif/schema.lua

```diff
@@ -1,8 +1,9 @@
 return {
   fields = {
     api_endpoint = {required = true, type = "url", default = "https://api.moesif.net"},
-    timeout = {default = 10000, type = "number"},
-    keepalive = {default = 10000, type = "number"},
+    timeout = {default = 1000, type = "number"},
+    keepalive = {default = 2000, type = "number"},
+    event_queue_size = {default = 2000, type = "number"},
```

Comment:
assuming avg request is 10Kb. then may be 5000 *10Kb is 50Mb is acceptable I think.. @dgilling , lemme know if u have any thoughts on default values of event_queue_size and timeouts above.

Reply:
lgtm. 10kb is the number I've always been using for typical REST or RPC API. We've only seen it larger if customer does something weird like try and log files or HTML with Moesif (technically an unsupported use case)